### DISCUSSION TITLE
Using git tags in release pipeline

### DISCUSSION THREAD

**Discussion by serge-koshelev** [rfennell/AzurePipelines#949] [Q&A] [answered]
Dear Sir,
Thank you for your pluging. It is very handy. I've got few times through documentation
but can't find example how to work with git tags.
Is it possible to generate release notes in release pipeline using the given tag from a git history?
Diff from the given tag till the latest git commit. It is more handy than using just release ID.

Kind regards
Serge

**Comment by rfennell** [comment, marked as answer]
There is no support for finding the last good build using Git Tags. The tags mentioned in the documentation are Azure DevOps Pipeline Tags.

One option is to add you tag to both Git and Azure DevOps. You can add the latter using this [extension](https://marketplace.visualstudio.com/items?itemName=colinsalmcorner.colinsalmcorner-buildtasks)
